repository-data/webfiles/src/main/resources/site/freemarker/intranet/news-internal.ftl

```diff
@@ -0,0 +1,126 @@
+                        </#list>
+                    </div>
+                </#if>
+            </div>
+
+            <#if hasLatestNews>
+                <div class="column column--one-third">
+                    <div class="news-latest-news">
+                        <div class="news-latest-news-title bottom-margin-20"><@fmt.message key="headers.latestNews" /></div>
+                        <ul class="intra-home-article-grid">
+                            <#list document.latestNews as news>
+                                <#assign hasShortSummary = news.shortsummary?? && news.shortsummary?has_content />
+
+                                <li class="intra-home-article-grid-item intra-home-article-grid-item--full">
+                                    <article class="intra-home-article-grid-article">
+                                        <div class="intra-home-article-grid-article__header">
+                                            <span class="intra-info-tag">News</span>
+                                            <#if news.leadImageSection?has_content && news.leadImageSection.leadImage?has_content>
+                                                <@hst.link hippobean=news.leadImageSection.leadImage.original fullyQualified=true var="leadImage" />
+                                                <img src="${leadImage}" alt="<#if hasLeadImageAltText>${news.leadImageSection.alttext}</#if>" />
+                                            <#else>
+                                                <img src="<@hst.webfile path="/images/fibre_57101102_med.jpg"/>" alt="Lead image for ${news.title}" aria-hidden="true">
+                                            </#if>
+                                        </div>
+                                        <div class="intra-home-article-grid-article__contents">
+                                            <h1 class="intra-home-article-grid-article__title">
+                                                <#assign onClickMethodCall = getOnClickMethodCall(document.class.name, news.title) />
+                                                <a href="<@hst.link hippobean=news/>" onClick="${onClickMethodCall}" onKeyUp="return vjsu.onKeyUp(event)">${news.title}</a>
+                                            </h1>
+
+                                            <#if hasShortSummary>
+                                                <div id="latest-article-${news?index}" class="intra-home-article-grid-article__summary">
+                                                    ${news.shortsummary}
+                                                </div>
+                                                <#-- call function webkitLineClamp(element, lineCount, colour) to implement multiline ellipsis -->
+                                                <script>
+                                                    webkitLineClamp(document.getElementById("latest-article-${news?index}"), 3, "black");
```

Comment:
What does this do?

Reply:
I believe it truncated the content to 3 lines